src/components/product/Product.jsx

```diff
@@ -40,7 +40,7 @@ const Product = ({
       <ProductQuantityInput
         readOnly={readOnly}
         defaultValue={qty}
-        onChange={(v) => { updateQuantity(v, lineItemKey); }}
+        onChange={(value) => { updateQuantity(lineItemKey, value); }}
```

Comment:
Not sure if this is an issue but I noticed the arguments provided to the updateQuantity function are flipped (lineItemKey is first now instead of second argument provided).

Reply:
Yup, This was changed to keep this consistent with the order of arguments that other hooks use. Also renamed v to be something more clear.